pkg/syncer/types.go

```diff
@@ -100,8 +100,11 @@ type (
 )
 
 type metadataSyncInformer struct {
-	clusterFlavor      cnstypes.CnsClusterFlavor
-	volumeManager      volumes.Manager
+	clusterFlavor cnstypes.CnsClusterFlavor
+	volumeManager volumes.Manager
+	// map of VC Host to Volume Manager
+	// Use this for Vanilla Multi vCenter Topology feature
```

Comment:
Can we instead say `// Use this for Vanilla flavor` as we will use this for single VC when FSS is removed

Reply:
done.